docs/installation/integrations/react-multiroot.md

```diff
@@ -0,0 +1,139 @@
+---
+menu-title: Multi-root integration
+meta-title: React rich text editor component | CKEditor 5 documentation
+category: react
+order: 20
+---
+
+{@snippet installation/integrations/framework-integration}
+
+# React rich text multi-root editor hook
+
+<p>
+	<a href="https://www.npmjs.com/package/@ckeditor/ckeditor5-react" target="_blank" rel="noopener">
+		<img src="https://badge.fury.io/js/%40ckeditor%2Fckeditor5-react.svg" alt="npm version" loading="lazy">
+	</a>
+</p>
+
+CKEditor&nbsp;5 consists of {@link installation/getting-started/predefined-builds ready-to-use editor builds} and {@link framework/index CKEditor&nbsp;5 Framework} upon which the builds are based.
+
+The easiest way to use multi-root CKEditor&nbsp;5 in your React application is by using the {@link installation/getting-started/predefined-builds#multi-root-editor multi-root rich text editor build}. You can also use a customized editor built by using [CKEditor&nbsp;5 online builder](https://ckeditor.com/ckeditor-5/online-builder/) in any React application.
```

Comment:
We don't have multi-root in the online builder.

Reply:
This whole intro seems confusing / unnecessary. I take it that it is a copy-paste from another guide, but maybe let's think if it makes sense here. Why the user should read it? What info the user gets from it? Do we want to redirect the user or show they a better/different approach?

This is a guide for a specific, multi-root integration. I'd assume that the user knows what they are doing if they are here. It may make more sense to give here some hint that this is for multi-root editor that supports multiple editable areas. If they are looking for a classic integration, they should look at the regular React integration. This will help people who get here straight from Google, for example.

Reply:
Wonder how will we do it with the New Installation Methods...